packages/core/src/components/link/link.stories.js

```diff
@@ -0,0 +1,92 @@
+import './link.css';
+import '../../icons/icon.css';
+
+export default {
+  title: 'Components/Link',
+  decorators: [(storyFn) => `<div style="max-width: 400px">${storyFn()}</div>`],
+};
+
+export const Default = () => `
+  <a href="/?path=/story/components-link--all" class="hds-link hds-link--small">Default link</a>
+`;
+
+
+export const InternalLinks = () => `
+  <a href="/?path=/story/components-link--all" class="hds-link hds-link--small">Internal link size small</a>
+  <br/>
+  <br/>
+  <a href="/?path=/story/components-link--all" class="hds-link hds-link--medium">Internal link size medium</a>
+  <br/>
+  <br/>
+  <a href="/?path=/story/components-link--all" class="hds-link hds-link--large">Internal link size large</a>
+`;
+
+InternalLinks.storyName = 'Internal links';
+
+export const ExternalLinks = () => `
+  <a href="https://hds.hel.fi" class="hds-link hds-link--small" aria-label="External link size small. Avautuu uuteen domainiin">
```

Comment:
If the link text is in English, the aria-label should also be fully in English. Also "Avautuu uuteen domainiin" probably doesn't tell anything for regular user. 
How about in Finnish "Siirtyy toiseen sivustoon." and in English "Opens a different website." In Swedish "Flyttar till en annan webbplats."

Reply:
- [ ] Improve aria-labels